lib/scenic/view.rb

```diff
@@ -14,11 +14,11 @@ def ==(other)
     end
 
     def to_schema
-      <<-DEFINITION.strip_heredoc
-        create_view :#{name}, sql_definition:<<-\SQL
-          #{definition}
-        SQL
+      <<-DEFINITION
 
+  create_view :#{name}, sql_definition: <<-\SQL
+    #{definition.indent(2)}
```

Comment:
I'd expect `SELECT` and `UNION` to be indented to the same level, making them cocolumnar with `create_view`. I'll leave it up to you whether we should then bump up the `indent` level to 4 to appear as a "body" to the call.